### PR TITLE
Add basic skeleton

### PR DESCRIPTION
**Pull request by gsvr**

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2014, Praekelt Foundation
```
Comment (imsickofmaps) [inline LICENSE]: 2015

File: Gruntfile.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    grunt.loadNpmTasks('grunt-contrib-jshint');
+    grunt.loadNpmTasks('grunt-mocha-test');
+    grunt.loadNpmTasks('grunt-contrib-concat');
+    grunt.loadNpmTasks('grunt-contrib-watch');
+
+    grunt.initConfig({
+        paths: {
+            src: {
+                app: {
+                    app1: 'src/app1.js',
```
Comment (imsickofmaps) [inline Gruntfile.js]: can we rename them to something more understandable? `voice_reg_and_change.js` and `sms_inbound.js`

File: src/utils.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    },
+
+    return_q: function() {
+        return Q();
+    },
+
+    control_api_call: function (method, params, payload, endpoint, im) {
+        var api = new JsonApi(im, {
+            headers: {
+                'Authorization': 'Token ' + im.config.control.api_key,
```
Comment (imsickofmaps) [inline src/utils.js:23]: I'm not sure `im.config` is available - I'm checking

File: test/fixtures.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+                'method': 'POST',
+                'headers': {
+                    'Authorization': ['Token test_key'],
+                    'Content-Type': ['application/json']
+                },
+                'url': 'http://localhost:8000/api/v1/contacts/',
+                'data': {
+                    "details": {
+                        "name": "Semi-Test",
+                        "msisdn": "082001"
```
Comment (imsickofmaps) [inline test/fixtures.js]: Wrong format - see https://github.com/praekelt/mama-ng-control/blob/develop/mama_ng_control/apps/contacts/tests.py#L104-L110

File: Gruntfile.js (hunk 2 above)
Comment (gsvr) [inline Gruntfile.js]: Sorry, forgot to change it after saving the generic project folder.

File: test/fixtures.js (hunk 4 above)
Comment (gsvr) [inline test/fixtures.js]: Was going to handle that later, but I've added it.

File: src/utils.js (hunk 3 above)
Comment (imsickofmaps) [inline src/utils.js:23]: It's not at the moment but we'll update later once they find a solution

**Comment by imsickofmaps** [comment]
:+1:

**Pull request merged**